### ISSUE TITLE
October 9th, 2018 Meeting

### ISSUE THREAD

**Issue by Blue-speed** [HRNUG/Organization#7]
Date: October 9th, 2018
- [x] Speaker
- [x] Sponsor
- [ ] Create meetup page

**Comment by TMarsdenMarathon** [comment]
Marathon can sponsor this one.

**Comment by jamessdixon** [comment]
I can speak if wanted:
Stock Cars, Raspberry PI, and F# FTCF (For the Checked Flag)

In North Carolina, stock car racing boarders on obsession – with local track racing rivaling high school football as a popular Friday night family activity. Different local tracks have different classes of cars and my son runs in the intro division of 5/8 size car at three local speedways. 

In this talk, I will cover: 
1. A brief introduction to oval track racing - and why it is much more than "go fast and turn left"
2. How we used a Raspberry PI to help capture some of the telemetry on the car 

How we used F# to analyze the data to help him go faster

BIO
https://mvp.microsoft.com/en-us/PublicProfile/5000814?fullName=Jamie
